### PR TITLE
Access key addition to access exchange rate in ENTSO-E implementation

### PR DESCRIPTION
**Pull request by venu-sagar**
Added access key configuration to suit the change in the access requirements for exchange rate host API.

### CODE REVIEW

**Comment by crvln** [comment]
Tested the fix and confirm it working!

File: io.openems.edge.timeofusetariff.entsoe/test/io/openems/edge/timeofusetariff/entsoe/ExchangeRateApiTest.java [hunk 1]
```diff
@@ -16,199 +15,39 @@ public class ExchangeRateApiTest {
...
+		var currency = Currency.SEK;
 		var response = Utils.exchangeRateParser(EXCHANGE_DATA, currency);
 
-		assertTrue(response == 1.0);
-
-		currency = Currency.SEK;
-		response = Utils.exchangeRateParser(EXCHANGE_DATA, currency);
-
-		assertFalse(response == 1.0);
+		assertTrue(response == 11.562427);
```
Comment (sfeilmeier) [inline io.openems.edge.timeofusetariff.entsoe/test/io/openems/edge/timeofusetariff/entsoe/ExchangeRateApiTest.java:51]: Float or Double values should never be compared like this. Use `assertEquals(11.562427, rate, 0.0001);` instead.

**Comment by sfeilmeier** [comment]
Thanks @crvln for testing. 

I am merging this to our local FEMS dev branch and will have to do some modifications. I will backport everything once its finished.

**Comment by sfeilmeier** [comment]
This has been solved in https://github.com/OpenEMS/openems/pull/2435. Closing here.

**Pull request closed without merging**